pandas/tests/extension/test_interval.py

```diff
@@ -95,7 +95,10 @@ class TestGrouping(BaseInterval, base.BaseGroupbyTests):
 
 
 class TestInterface(BaseInterval, base.BaseInterfaceTests):
-    pass
+    def test_view(self, data):
```

Comment:
can't these tests be in the base class?

Reply:
they are in the base class.  we have to override it here because `IntervalArray.__setitem__` incorrectly makes a copy (#27147)